python/composio/tools/toolset.py

```diff
@@ -67,22 +67,28 @@
 P = te.ParamSpec("P")
 
 _KeyType = t.Union[AppType, ActionType]
-_ProcessorType = t.Callable[[t.Dict], t.Dict]
+_CallableType = t.Callable[[t.Dict], t.Dict]
 
 MetadataType = t.Dict[_KeyType, t.Dict]
 ParamType = t.TypeVar("ParamType")
 
+# Enable deprecation warnings
+warnings.simplefilter("always", DeprecationWarning)
+
+
+ProcessorType = te.Literal["pre", "post", "schema"]
```

Comment:
The `ProcessorType` type alias is defined but not used in this file. Consider removing it if it's not needed, or use it to improve type checking in the relevant parts of the code.